### PR TITLE
Adds storybook integrity check to drone config

### PR DESCRIPTION
**Pull request by dprokop**
To be merged after https://github.com/grafana/build-pipeline/pull/150

### CODE REVIEW

**Review by aknuds1 — CHANGES_REQUESTED** [review]
You will need to make the changes in the Starlark code (scripts/*.star), since .drone.yml is auto-generated from that.

File: scripts/lib.star [hunk 1]
```diff
@@ -272,6 +272,23 @@ def build_storybook_step(edition, ver_mode):
         ],
     }
 
+def verify_storybook_step(edition, ver_mode):
```
Comment (aknuds1) [inline scripts/lib.star:275]: Could you include the command `./bin/grabpl verify-storybook' in the build-storybook and publish-storybook steps instead? I don't see any benefit to making its own step. Every step introduces some risk/complexity/performance degradation.

**Review by aknuds1 — COMMENTED** [review]
You will also need to bump grabpl to 0.5.24, since I just uploaded that.

File: scripts/lib.star [hunk 2]
```diff
@@ -298,7 +299,7 @@ def publish_storybook_step(edition, ver_mode):
         'name': 'publish-storybook',
         'image': publish_image,
         'depends_on': [
-            'build-storybook',
+            'verify-storybook',
```
Comment (torkelo) [inline scripts/lib.star:302]: Is this right? it looks like it's part of the build step, not a seperate drone step

Comment (aknuds1) [reply scripts/lib.star]: @torkelo You're right, that change shouldn't be made. I fixed it.

**Review by aknuds1 — APPROVED** [review]

**Comment by aknuds1** [comment]
Great! :)

File: scripts/lib.star (hunk 2 above)
Comment (dprokop) [reply scripts/lib.star]: thanks @aknuds1

**Pull request merged**